### PR TITLE
updated team profile

### PR DESCRIPTION
**Pull request by nams05**

### CODE REVIEW

File: team.html [hunk 1]
```diff
@@ -477,7 +547,7 @@ <h2 class="section-head">Our Team</h2>
 
 <footer>
     <div class="container">
-    <br>
+    <br>t
         <div class="row">
```
Comment (kartikjain) [inline team.html:551]: why this t?

File: partners.html [hunk 2]
```diff
@@ -168,7 +168,7 @@ <h2 class="section-head">Partners</h2>
     <script src="js/bootstrap.min.js"></script>
     <script src="js/bootstrap-select.js"></script>
     <script src="js/bootstrap-switch.js"></script>
-    <script src="js/app.subscribe.js" type="text/javascript"></script>
+    <script src="js/subscribe.js" type="text/javascript"></script>
     <script src="js/contentHover.js"></script>
```
Comment (kartikjain) [inline partners.html:172]: plz explain the changes in this file

File: team.html (hunk 1 above)
Comment (nams05) [inline team.html:551]: sorry ...i dont know how it came there...i'll fix it

File: partners.html (hunk 2 above)
Comment (nams05) [inline partners.html:172]: i'll fix this too...the file shudnt be there.

File: team.html [hunk 3]
```diff
@@ -197,7 +200,8 @@ <h2 class="section-head">Our Team</h2>
                 <div class="ch-info">
                     <div class="ch-info-front ch-harshita"></div>
                     <div class="ch-info-back">
-                        <h4>Hortative<br>High-spirited<br>Hardy <br>Happy-go-Lucky <br>Humanitarian
+                        <h4>Keen Educator<br>Quick Learner<br> Creator<br> Innovator<br>Easily Motivated
+
                         </h4>
```
Comment (kartikjain) [inline team.html:205]: please remove the extra line

File: team.html [hunk 4]
```diff
@@ -393,7 +397,7 @@ <h2 class="section-head">Our Team</h2>
                 <div class="ch-info">
                     <div class="ch-info-front ch-geetika"></div>
                     <div class="ch-info-back">
-                        <h4>Cinephilia<br>Tech freak <br>Budding filmmaker <br>Quirky<br>Travel Buff
+                        <h4>Cinephilia<br>Tech freak <br>Budding filmmaker <br>Quirky<br>Trained bathroom singer
```
Comment (kartikjain) [inline team.html:400]: please verify if we still have this person on team

**Pull request closed without merging**